src/tv2_afvd_showstyle/postProcessTimelineObjects.ts

```diff
@@ -165,7 +167,7 @@ export function postProcessPieceTimelineObjects(
 
 				const cleanObj = literal<TSR.TimelineObjAtemME & TimelineBlueprintExt>({
 					..._.omit(tlObj, 'content'),
-					...literal<Partial<TSR.TimelineObjAtemME & TimelineBlueprintExt>>({
+					...literal<Partial<TSR.TimelineObjAtemME & TimelineBlueprintExt> & Pick<TSR.TimelineObjAtemME, 'content'>>({
```

Comment:
It complains that TimelineContentTypeAtem.DSK is incompatiable with TimelineContentTypeAtem.ME if we make the same syntax here, as your comment above.
Not sure how to get rid of it if not using this solution

Reply:
Then do `..._.omit(tlObj, 'content', 'keyframes')` because `keyframes` are taking their type from `content` and we neither use them, nor would it make sense to copy them here to a different type of timeline object.

Reply:
Nice :) I've updated the PR